### ISSUE TITLE
Padrão dos componentes

### ISSUE THREAD

**Issue by lhguerra** [gustavogodoi/taller-app#9]
Reparei que alguns componentes você escreveu como classe (extendendo o Component do React) e outros como função, simplesmente retornando um JSX. Você saberia explicar a diferença entre as duas abordagens e por que escolheu fazer diferente em alguns casos?

**Comment by gustavogodoi** [comment]
Olá @lhguerra , para alguns components mais simples, criei como `Stateless Component`. Quando não precisamos de nenuma função do `React.Component` (constructor, componentWillMount, ComponentDidMount, etc) ou `state` é a melhor opção por ser mais performático.

**Comment by lhguerra** [comment]
Entendi. Mais uma pergunta só pra complementar: você conhece alguma forma de fazer todos os componentes `stateless`?

**Comment by gustavogodoi** [comment]
Sim, eu poderia transformar meus components em `Containers` e `Stateless`. Onde toda a lógica fica no `Containers` e o `Stateless` carrega o HTML e o CSS (como um arquivo de template), recebendo apenas as `props` (variáveis ou funções de callback)
